va/va_enc_av1.h

```diff
@@ -0,0 +1,974 @@
+         * (segment_feature_support & 0x40) == 1: SEG_LVL_SKIP is supported, 0: not.
+         * (segment_feature_support & 0x80) == 1: SEG_LVL_GLOBALMV is supported, 0: not.
+         */
+        uint32_t segment_feature_support       : 8;
+        /** \brief Reserved bits for future, must be zero. */
+        uint32_t reserved                      : 11;
+    } bits;
+    uint32_t value;
+} VAConfigAttribValEncAV1Ext1;
+
+/** \brief Attribute value for VAConfigAttribEncAV1Ext2. */
+typedef union _VAConfigAttribValEncAV1Ext2 {
+    struct {
+        /**
+        * \brief Tile size bytes minus1.
+        * Specify the number of bytes needed to code tile size supported.
+        * This value need to be set in frame header obu.
+        */
+        uint32_t tile_size_bytes_minus1        : 2;
+        /**
+        * \brief Tile size bytes minus1.
+        * Specify the fixed number of bytes needed to code syntax obu_size.
+        */
+        uint32_t obu_size_bytes_minus1         : 2;
+        /**
+         * \brief tx_mode supported.
+         * (tx_mode_supported & 0x01) == 1: ONLY_4X4 is supported, 0: not.
+         * (tx_mode_supported & 0x02) == 1: TX_MODE_LARGEST is supported, 0: not.
+         * (tx_mode_supported & 0x04) == 1: TX_MODE_SELECT is supported, 0: not.
+         */
+        uint32_t tx_mode_supported             : 3;
+        /**
+         * \brief Max tile num minus1.
+         * Specify the max number of tile supported by driver.
+         */
+        uint32_t max_tile_num_minus1           : 13;
+        /** \brief Reserved bits for future, must be zero. */
+        uint32_t reserved                      : 12;
+    } bits;
+    uint32_t value;
+} VAConfigAttribValEncAV1Ext2;
+
+/**
+ * \brief Packed header types specific to AV1 encoding.
+ *
+ * Types of packed headers generally used for AV1 encoding.
+ *
+ */
+typedef enum {
+    /**
+     * \brief Packed Sequence Parameter Set (SPS).
+     *
+     * The corresponding packed header data buffer shall contain the
+     * complete sequence_header_obu() syntax element.
+     *
+     */
+    VAEncPackedHeaderAV1_SPS = VAEncPackedHeaderSequence,
+    /**
+     * \brief Packed Picture Parameter Set (PPS).
+     *
+     * The corresponding packed header data buffer shall contain the
+     * complete frame_header_obu() syntax element.
+     *
+     */
+    VAEncPackedHeaderAV1_PPS = VAEncPackedHeaderPicture,
+} VAEncPackedHeaderTypeAV1;
+
+/**
+ * \brief AV1 Encoding Sequence Parameter Buffer Structure.
+ *
+ * This structure conveys sequence level parameters.
+ *
+ */
+typedef struct  _VAEncSequenceParameterBufferAV1 {
+    /** \brief AV1 profile setting.
+     *  value range [0..2].
+     */
+    uint8_t     seq_profile;
+
+    /** \brief Level Setting of current operation point.
+     *  value range [0..23].
+     */
+    uint8_t     seq_level_idx;
+
+    /** \brief Tier Setting of current operation point.
+     *  value range [0..1].
+     */
+    uint8_t     seq_tier;
```

Comment:
what's tier? could not find the key words in spec.

Reply:
there will be seq_tier in spec, which specify the tier that the coded video sequence conforms to.  seq_tier will constrain MaxBitrate/MainCR at certain profile/level.